docs/guides/cloud/test-analytics/analytics.mdx

```diff
@@ -1,6 +1,7 @@
 ---
-title: Analytics
-sidebar_position: 60
+title: Test Suite Analytics
```

Comment:
per yesterday's discussion, flatten to top-level and name as Analytics

Reply:
Which of options that we considered late in the meeting did this? I must have missed it.

Reply:
I'm working on pulling out the notes & will send your way when I'm done!